### PR TITLE
feat: add fork helper scripts

### PR DESCRIPTION
**Pull request by skhomuti**
first one is simulating vote

- removed MODULE_ID because it's difficult to get module id from the script and pass it to the tests, for loop search seems ok for me
- staking router interface updated to the new version
- removed KEEP_ANVIL_AFTER_LOCAL_DEPLOY env var because it seems redundant

### CODE REVIEW

**Review by dgusakov — APPROVED** [review]

File: .env.sample [hunk 1]
```diff
@@ -1,7 +1,6 @@
 RPC_URL=
```
Comment (dgusakov) [inline .env.sample:1]: Add `RPC_URL_HOLESKY`

File: Justfile [hunk 2]
```diff
@@ -121,5 +121,11 @@ test-local *args:
         just test-integration {{args}}
     just kill-fork
 
+simulate-vote:
+    cast rpc anvil_autoImpersonateAccount true
```
Comment (madlabman) [inline Justfile:125]: I think we should provide the exact rpc-url here.

File: script/fork-helpers/SimulateVote.sol [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+        IStakingRouter stakingRouter = IStakingRouter(locator.stakingRouter());
+        IBurner burner = IBurner(locator.burner());
+
+        address agent = stakingRouter.getRoleMember(
+            stakingRouter.STAKING_MODULE_MANAGE_ROLE(),
+            0
+        );
+        vm.label(agent, "agent");
+        console.logUint(agent.balance);
```
Comment (madlabman) [inline script/fork-helpers/SimulateVote.sol:25]: I assume we need for a check, and not just a logging.

File: .env.sample (hunk 1 above)
Comment (skhomuti) [reply .env.sample:1]: added a note, no need to change vars here

File: script/fork-helpers/SimulateVote.sol [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+        vm.rpc(
+            "anvil_setBalance",
+            string.concat(
+                '["',
+                vm.toString(csmAdmin),
+                '", ',
+                "1000000000000000000",
+                "]"
+            )
+        );
```
Comment (madlabman) [inline script/fork-helpers/SimulateVote.sol:46]: What about to make it a function?

File: test/integration/Deploy.t.sol [hunk 5]
```diff
@@ -25,7 +25,7 @@ contract TestDeployment is Test, DeploymentFixtures {
         vm.startPrank(csm.getRoleMember(csm.DEFAULT_ADMIN_ROLE(), 0));
         csm.grantRole(csm.RESUME_ROLE(), address(this));
         vm.stopPrank();
-        csm.resume();
+        if (csm.isPaused()) csm.resume();
```
Comment (madlabman) [inline test/integration/Deploy.t.sol:28]: I would move the `grantRole` and the `prank` stuff in the if statement body to make it easier to read the trace.

File: test/integration/StakingRouter.t.sol [hunk 6]
```diff
@@ -54,24 +54,30 @@ contract StakingRouterIntegrationTest is Test, Utilities, DeploymentFixtures {
...
+            _priorityExitShareThreshold: 10000,
             _stakingModuleFee: 500,
-            _treasuryFee: 500
+            _treasuryFee: 500,
+            _maxDepositsPerBlock: 30,
+            _minDepositBlockDistance: 25
         });
-        uint256[] memory ids = stakingRouter.getStakingModuleIds();
-        return ids[ids.length - 1];
+        return ids[ids.length - 1] + 1;
```
Comment (madlabman) [inline test/integration/StakingRouter.t.sol:80]: Just `ids.length`?

File: script/fork-helpers/SimulateVote.sol (hunk 3 above)
Comment (skhomuti) [reply script/fork-helpers/SimulateVote.sol:25]: this is a debug thing actually, thx

**Review by dgusakov — DISMISSED** [review]

File: test/integration/Deploy.t.sol (hunk 5 above)
Comment (skhomuti) [reply test/integration/Deploy.t.sol:28]: added if this is the only role to be granted

File: Justfile (hunk 2 above)
Comment (madlabman) [reply Justfile:125]: Yes, but we have anvil on `{{ anvil_host }}:{{ anvil_port }}`

**Review by dgusakov — APPROVED** [review]

**Review by madlabman — APPROVED** [review]

**Pull request merged**